scripts/start-setupModpack

```diff
@@ -192,6 +194,12 @@ function handleGenericPacks() {
         extract "${pack}" "${base_dir}"
       done
 
+      # Disable mods
+      for mod in ${DISABLE_MODS}; do
+	log Disabling $mod
+	find "${base_dir}" -name "$mod" -exec mv {} {}.DISABLED -v \;
```

Comment:
Let's use a suffix of ".disabled", lowercase to be consistent with other launcher/solutions.